article/test/CdnHealthCheckTest.scala

```diff
@@ -14,7 +15,9 @@ import org.scalatest.concurrent.ScalaFutures
   with WithTestWsClient {
 
   "CDN health check" should "mimic the instance health check" in {
-    val controller = new HealthCheck(wsClient)
+    val testPort: Int = port
```

Comment:
Necessary, otherwise the `port` val in the constructor closure shadows the test `port`